### PR TITLE
Rename static/.../javascripts to static/.../js

### PR DESCRIPTION
**Pull request by orangejenny**
More `static` standardization.

@czue

### CODE REVIEW

File: custom/m4change/templates/m4change/fields/daterange.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
 {% endblock %}
 {% block filter_js %}
     <script type="text/javascript">
-        $.getScript("{% static 'm4change/javascripts/daterangepicker.js' %}", function () {
+        $.getScript("{% static 'm4change/js/daterangepicker.js' %}", function () {
```
Comment (czue) [inline custom/m4change/templates/m4change/fields/daterange.html]: might be missing something but it doesn't look like this file moves in this PR to me.

File: custom/m4change/templates/m4change/selectTemplate.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
 {% block js %} {{ block.super }}
     <script type="text/javascript" src="{% static 'case/js/cheapxml.js' %}"></script>
     <script type="text/javascript" src="{% static 'case/js/casexml.js' %}"></script>
-    <script type="text/javascript" src="{% static 'm4change/javascripts/mcct_project_review_page_management.js' %}"></script>
+    <script type="text/javascript" src="{% static 'm4change/js/mcct_project_review_page_management.js' %}"></script>
```
Comment (czue) [inline custom/m4change/templates/m4change/selectTemplate.html]: same with this one

**Review by czue — CHANGES_REQUESTED** [review]
waiting till you clear up whether there's a missing move in here

**Comment by orangejenny** [comment]
Didn't mean to update the references to m4change files, thanks for catching.

**Review by czue — APPROVED** [review]

**Pull request merged**